test/unittest/CMakeLists.txt

```diff
@@ -55,6 +55,10 @@ set(SYCL_UNITTEST_SRCS
   ${SYCLBLAS_UNITTEST}/blas3/blas3_gemm_batched_test.cpp
   ${SYCLBLAS_UNITTEST}/blas3/blas3_trsm_test.cpp
   ${SYCLBLAS_UNITTEST}/blas3/blas3_symm_test.cpp
+  # Blas extension
+  ${SYCLBLAS_UNITTEST}/extension/omatcopy_test.cpp
+  ${SYCLBLAS_UNITTEST}/extension/omatcopy2_test.cpp
+  ${SYCLBLAS_UNITTEST}/extension/transpose_test.cpp
```

Comment:
I think they should be moved below, inside the `if(${BLAS_ENABLE_EXTENSIONS})` statement, since they belong to the extensions.

Reply:
Thanks for highlighting this @pgorlani. We haven't fully included these operators in the "Extensions" group in SYCL BLAS because it doesn't match by definition the BLAS extensions defined by oneMKL spec or cuBLAS/rocBLAS. So far it has been limited to the reduction operator which is not an explicit operator in BLAS extensions as defined in the reference specs. So it's a bit confusing I agree to re-conciliate the extensions option in SYCL BLAS & the recently added extension operators, especially that there is no such flag in other libraries I guess. 
But anyways, we can put them all together (and disable their generation/build alongside tests & benchs as it's done with reduction) or separate the reduction operator flag if it has any property different from the extension ops I'm not aware of.

Reply:
I agree with @OuadiElfarouki that we probably need another flag to split these extensions and the one already present, but we can start moving them all together and separate the two later.